src/components/competition/charts/timeline-chart.tsx

```diff
@@ -81,7 +85,7 @@ class CompetitionTimeLineChart extends React.Component<Props, State> {
         const { data } = this.state;
         return (
             <div>
-                <ChartTitle>Uppökur per dag</ChartTitle>
+                <ChartTitle>Upptökur per dag</ChartTitle>
```

Comment:
I don't think per is an icelandic word unless I just don't know enough icelandic

Reply:
I think you are correct, should probably be "Upptökur á dag"